iaso/api/mobile/org_units.py

```diff
@@ -30,7 +32,34 @@ def get_page_number(self, request):
         return int(request.query_params.get(self.page_query_param, 1))
 
 
-class MobileOrgUnitSerializer(ModelSerializer):
+class ReferenceInstancesFilter(django_filters.rest_framework.FilterSet):
+    last_sync = django_filters.IsoDateTimeFilter(field_name="updated_at", lookup_expr="gte")
+
+    class Meta:
+        model = Instance
+        fields = []
+
+
+class ReferenceInstancesSerializer(serializers.ModelSerializer):
+    org_unit_id = serializers.IntegerField(source="org_unit.id")
+    org_unit_uuid = serializers.UUIDField(source="org_unit.uuid")
```

Comment:
I can remove `org_unit_id` and `org_unit_uuid` if you don't see any value.

Reply:
Yes, no problem to remove them